canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.jsx

```diff
@@ -89,6 +91,17 @@ class VirtualizedTable extends React.Component {
 		}
 	}
 
+	getCheckboxLabelColumnIndex(columns) {
+		// If 1st column is Index, use the next column for labeling
+		for (let i = 0; i < columns.length; i++) {
+			if (columns[i].headerLabel !== "Index") {
```

Comment:
Is there another way to check for the index column? We shouldn't compare this against the label. It wont work for translations as well. Are you able to see if the columns contain anything for `"operation": "index"`?